composer.json

```diff
@@ -4,9 +4,27 @@
   "license": "MIT",
   "type": "phpcodesniffer-standard",
   "require": {
-    "squizlabs/php_codesniffer": "^3.2"
+    "squizlabs/php_codesniffer": "^3.2",
+    "symplify/coding-standard": "dev-main",
+    "slevomat/coding-standard": "^8.15",
+    "friendsofphp/php-cs-fixer": "^3.64",
+    "symfony/dependency-injection": "^7.1",
+    "symfony/config": "^7.1"
   },
+  "repositories": [
+    {
+      "type": "vcs",
+      "url": "https://github.com/mdk999/coding-standard"
```

Comment:
should this be mdk999 or something else?

Reply:
this fork should be pulled into nocworx organization instead - @mpedrummer will take care of that

Reply:
Yeah - what's up with this? Shouldn't you be using `nocworx/phpcs` after we merge?